examples/overlays/add-stylized/index.js

```diff
@@ -0,0 +1,57 @@
+function removeAllOverlays(){
+    // Remove all overlays
+    // Shape
+    bpmnElementsRegistry.removeAllOverlays('Activity_1potg3p');
+
+    // Edge
+    bpmnElementsRegistry.removeAllOverlays('Flow_1wkfbb0');
+}
+
+
+// Initialize the panel of Overlay settings
+var fontColorElt = document.getElementById('font-color');
+var fontSizeElt = document.getElementById('font-size');
+var fillColorElt = document.getElementById('fill-color');
+var strokeColorElt = document.getElementById('stroke-color');
+
+var style = {
+    font: {
+        color: fontColorElt.value,
+        size: fontSizeElt.value,
+    },
+    fill: {
+        color: fillColorElt.value,
+    },
+    stroke: {
+        color: strokeColorElt.value,
+    }
+};
+
+fontColorElt.onchange = (event) => style.font.color = event.target.value;
+fontSizeElt.onchange = (event) => style.font.size = event.target.value;
+fillColorElt.onchange = (event) => style.fill.color = event.target.value;
+strokeColorElt.onchange = (event) => style.stroke.color = event.target.value;
+
+// Initialize BpmnVisualization
+const bpmnVisualization = new bpmnvisu.BpmnVisualization({ container: 'bpmn-container', navigation: { enabled: true } });
+const bpmnElementsRegistry = bpmnVisualization.bpmnElementsRegistry;
+
+// Load BPMN diagram
+bpmnVisualization.load(getGettingStartedBpmnDiagram(), { fit: { type: 'Center' } });
+
+document.getElementById('btn-set-overlay').onclick = () => {
+    removeAllOverlays();
+
+    // Add overlays
+    // Shape
+    bpmnElementsRegistry.addOverlays('Activity_1potg3p', { position: 'bottom-right', label: 'NaN', style });
```

Comment:
❓ `NaN` may be confusing as it often show an overfow. Can we use something else?

Reply:
Yes, but I don't have any idea. 
I just wanted to show that we can use **text** and not just **number** as value.

If you have a idea, you can change & merge 😉

Reply:
we could put`OK`? or something with an emoji as well?